yarn-project/aztec-cli/src/index.ts

```diff
@@ -20,6 +20,7 @@ import { ContractData, TxHash, L2BlockL2Logs } from '@aztec/types';
 
 import { encodeArgs, parseStructString } from './cli_encoder.js';
 import { deployAztecContracts, getContractAbi, getTxSender, prepTx } from './utils.js';
+import { SchnorrAccountContractAbi } from '../../noir-contracts/src/examples/index.js';
```

Comment:
Should this be imported via 'aztec/noir-contracts'?

Reply:
😳 VS Code betrayed me